iree_tests/README.md

```diff
@@ -199,3 +206,125 @@ python -m pip install iree-compiler iree-runtime
 python -m pip install -r ./iree_tests/onnx/requirements.txt
 python ./iree_tests/onnx/import_tests.py
 ```
+
+### PyTorch models
```

Comment:
Some other ideas once there are a few more models

* Run on CPU with `local-task` instead of `local-sync`, possibly setting other flags
* Qualify test case names in XFAIL sets based on their test package (`test_acos` should be more like `onnx/node/generated/test_acos/model.mlir`, can drop 'test_' too)
* Tag tests with markers (type of test, size of test, etc.) for easier filtering in CI jobs and on developer machines

Reply:
Could also log test durations with `--durations=` (https://stackoverflow.com/a/27899853)

On my machine: `10.51s call     pytorch/models/resnet50/resnet50.mlirbc::cpu_llvm_sync` ... not sure if that slow speed (for compile + run) is expected, needs different flags, or what.

For logs to be most useful, we could splitting into 'compile' and 'run' test cases and have a separate pytest invocation for "small" tests and "large" tests that choose different levels of logging to use.

Reply:
compile + run on the new tests, on a standard sized runner:
```
11.07s call     iree_tests/pytorch/models/resnet50/resnet50.mlirbc::cpu_llvm_sync
7.42s call     iree_tests/pytorch/models/opt-125M/opt-125M.mlirbc::cpu_llvm_sync
```